Doc/library/sqlite3.rst

```diff
@@ -1223,6 +1250,42 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      *autocommit* has three allowed values:
```

Comment:
```suggestion
      :attr:`!autocommit` has three allowed values:
```

We're taking about the attribute, not the parameter here, so we should format it accordingly.

Reply:
Good catch!